operator/pkg/components/iamauthenticator/iamauthenticator.go

```diff
@@ -0,0 +1,157 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package iamauthenticator
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"html/template"
+
+	"github.com/awslabs/kit/operator/pkg/utils/imageprovider"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/scheme"
+	"knative.dev/pkg/ptr"
+)
+
+func Config(ctx context.Context, name, ns, instanceRole, accountID string) (*v1.ConfigMap, error) {
```

Comment:
Optional, name it what it is:
```suggestion
func ConfigMap(ctx context.Context, name, ns, instanceRole, accountID string) (*v1.ConfigMap, error) {
```